l10n_us_form_1099/data/type_1099_data.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<odoo noupdate="1">
```

Comment:
@brian10048 I would remove the noupdate because this data should be managed by the module. If there is a new type or a type is renamed/merged with another, we update the module, release a new version and then anyone can upgrade.